src/Kafka/Consumer/KafkaConsumerBuilder.php

```
@@ -180,6 +200,9 @@ public function setRebalanceCallback(callable $rebalanceCallback): KafkaConsumer
     }
 
     /**
+     * Only applicable for the high level consumer
```

Maybe throw an exception if one wants to set it for low level?